packages/server/src/course/course.controller.ts

```diff
@@ -73,4 +74,22 @@ export class CourseController {
 
     return queue;
   }
+
+  @Delete(':id/ta_location/:room')
+  async checkOut(
+    @Param('id') courseId: number,
+    @Param('room') room: string,
+    @User() user: UserModel,
+  ): Promise<void> {
+    const queue = await QueueModel.findOne(
+      {
+        room,
+        courseId,
+      },
+      { relations: ['staffList'] },
+    );
+
```

Comment:
If the ta is not in the current `staffList` for the queue should we return a response so the front end knows the user wasn't actually checked out?

Reply:
I think it's fine cause either way the TA isn't in the queue